_posts/2017/06/2017-06-06-workshop-access.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+authors: ["Erin Becker, Jonah Duckles, Kari L. Jordan, Maneesha Sane, Tracy Teal"]
```

Comment:
Please use a list of strings rather than putting all names in a single string: `["Erin Becker", "Jonah Duckles", ...]`